x/stake/validator.go

```diff
@@ -154,6 +155,21 @@ func (v Validator) UpdateStatus(pool Pool, NewStatus sdk.BondStatus) (Validator,
 	return v, pool
 }
 
+// Remove & burn pool shares, e.g. when slashing a validator
+func (v Validator) removePoolShares(pool Pool, amt sdk.Rat) (Validator, Pool, int64) {
```

Comment:
`amt` is an ambiguous variable name, let's use `poolShares`.

Reply:
Also the comment is not exactly true - because this function returns int64 of the tokens, it actually implies that these tokens could be redistributed by the higher level logic that is calling this function. I think this capability is positive, there has been talk before about redistributing some of the tokens to victims of potential attacks - maybe let's add return variable names too for clarity

Reply:
Changed to `poolShares`, clarified comment.